Dockerfile

```diff
@@ -1,32 +1,62 @@
 #
```

Comment:
yep , I thought about it too, but there is currently no "import" for Dockerfiles :)